app/components/participant_status_tag_component.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ParticipantStatusTagComponent < BaseComponent
+  def initialize(profile:, admin: false)
+    @profile = profile
+    @admin = admin
+  end
+
+  def call
+    if profile.npq?
+      render Admin::Participants::NPQValidationStatusTag.new(profile: profile)
+    else
+      govuk_tag tag_attributes
+    end
+  end
+
+private
+
+  attr_reader :admin, :profile
+
+  def tag_attributes
+    return { text: "DfE to contact participant", colour: "grey" } unless FeatureFlag.active?(:participant_validation, for: profile.school_cohort.school)
+    return { text: "Manual checks needed", colour: "turquoise" } if admin && manual_check_needed
+    return { text: "DfE checking eligibility", colour: "blue" } if profile.ecf_participant_validation_data.present?
+
+    { text: "DfE requested details from participant", colour: "yellow" }
+  end
+
+  def manual_check_needed
+    profile.ecf_participant_eligibility&.manual_check_status? ||
+      (profile.ecf_participant_validation_data.present? && profile.ecf_participant_eligibility.nil?)
+  end
```

Comment:
Does this want to live on the profile?

Reply:
Hmm, maybe. Most of this sort of things lives on the eligibility record, but this obviously can't. Feels a little odd having it on the profile, but probably less odd than exposing it here